src/main/java/io/vertx/ext/hawkular/impl/inventory/InventoryReporter.java

```diff
@@ -114,7 +101,16 @@ public void report() {
       });
     });
   }
+
   public void stop() {
     httpClient.close();
   }
+
+  public InventoryReporter setHttpSocketAddresses(Set<SocketAddress> httpSocketAddresses) {
+    this.httpSocketAddresses = httpSocketAddresses;
+    for (SocketAddress addr : httpSocketAddresses) {
+      subResourceReporters.add(new HttpServerResourceReporter(options, httpClient, addr));
```

Comment:
@tsegismont I tried to add all http servers here. But they are in different context so `httpSocketAddresses` would be empty when it's passed. How can I handle this?